src/Umbraco.Web/Security/AppBuilderExtensions.cs

```diff
@@ -197,11 +152,11 @@ public static class AppBuilderExtensions
                 // Enables the application to validate the security stamp when the user
                 // logs in. This is a security feature which is used when you
                 // change a password or add an external login to your account.
-                OnValidateIdentity = SecurityStampValidator
-                    .OnValidateIdentity<BackOfficeUserManager, BackOfficeIdentityUser, int>(
-                        TimeSpan.FromMinutes(30),
-                        (manager, user) => manager.GenerateUserIdentityAsync(user),
-                        identity => identity.GetUserId<int>()),
+                OnValidateIdentity = UmbracoSecurityStampValidator
+                    .OnValidateIdentity<BackOfficeSignInManager, BackOfficeUserManager, BackOfficeIdentityUser>(
+                        TimeSpan.FromMinutes(3),
```

Comment:
Changed from 30 to 3. Would this be something to put in config?

Reply:
Good catch. Some test settings left in 🙄